help/desktop-app-install-guide.md

```diff
@@ -98,6 +98,22 @@ automatically to future versions.
 
 3. Run Zulip from your app launcher, or with `zulip` from a terminal.
 
+#### Flathub
+
+1. Make sure you have [Flatpak](https://flathub.org/setup) installed on your system.
+
+2. Use the following command to install Zulip from Flathub:
+
+        flatpak install flathub org.zulip.Zulip
+
+
+3. After the installation is complete, you can run Zulip using the following command:
+
+        flatpak run org.zulip.Zulip
+
+
+You can always check the official [Flathub page](https://flathub.org/apps/org.zulip.Zulip) for installation.
+
 <!-- TODO why dpkg? -->
```

Comment:
The formatting of this instructions block needs to be tweaked a bit. There shouldn't be any extra blank lines before or after each step,  and each line shouldn't exceed 80 characters. Also, it's probably safe to remove the `<!-- TODO why dpkg? -->` comment which seems to have been left over a while back.